eva-accession-cluster/pom.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>eva-accession</artifactId>
+        <groupId>uk.ac.ebi.eva</groupId>
+        <version>0.3.2</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>eva-accession-cluster</artifactId>
```

Comment:
Judgement call: May be we should name the package **eva-accession-clustering** to be more clear?

Reply:
I thought that too, but at this point the word has lost its meaning to me, out of repetition.

Reply:
We will still know what it means... I meant for someone new who is looking at the code :D

Reply:
Module renamed to eva-accession-clustering